corehq/messaging/scheduling/views.py

```diff
@@ -115,6 +249,7 @@ def possible_sms_recipients(request, domain):
     # TODO Add locations
     # TODO Add mobile worker groups
     # TODO will need to know doc type as well
+    # TODO Support "send to all"
```

Comment:
(we can leave out support for send to all)

Reply:
Thanks I missed this comment earlier https://github.com/dimagi/commcare-hq/pull/17722/commits/5ddb89e5b8a08962c73deb40c9623eaf7a1423fc